sherpa.spec

```diff
@@ -5,6 +5,10 @@
 Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
 Requires: hepmc lhapdf blackhat sqlite fastjet openssl
 
+%if "%(case %cmsplatf in (*amd64*) echo true ;; (*) echo false ;; esac)" == "true"
```

Comment:
Indeed. We should really have something like `linux_x86_64_glibcXX` 
because in the end that's what we really need to know for compatibility. 
This is a separate issue, though.



On 19 Jan 2015, at 16:02, davidlt wrote:

>> @@ -5,6 +5,10 @@
>> Source: 
>> git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
>> Requires: hepmc lhapdf blackhat sqlite fastjet openssl
>>
>> +%if "%(case %cmsplatf in (*amd64*) echo true ;; (*) echo false ;; 
>> esac)" == "true"
>
> Ah, I see. Our triplets aren't that good, missing "linux", "darwin", 
> "freebsd", etc. We don't have a type of OS.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-sw/cmsdist/pull/1295/files#r23164243